charts/vh-test-api/values.demo.template.yaml

```diff
@@ -0,0 +1,7 @@
+---
+java:
+  image: '${IMAGE_NAME}'
```

Comment:
can you add the two lines here please, just for Demo.

  ingressClass: traefik-private
  disableTraefikTls: false